### ISSUE TITLE
Documentation: Include Fabric component in all examples

### ISSUE THREAD

**Issue by scriby** [OfficeDev/office-ui-fabric-react#1450]
When tabbing into a toggle control, there is no visual indication that it has received focus. This violates Microsoft Accessibility Standard 05A.

**Comment by c-w** [comment]
I found that you can fix the lack of focus indicator by wrapping your app inside of a `<Fabric>` [component](https://github.com/OfficeDev/office-ui-fabric-react/blob/7ae401c36debef2a4700372b12fd6ff63b817f34/packages/office-ui-fabric-react/src/components/Fabric/Fabric.tsx#L33) which manages global state like "which component currently has focus".

Maybe the need to wrap the app inside of `<Fabric>` should be called out in the documentation/usage examples? (E.g. [here](https://github.com/OfficeDev/office-ui-fabric-react#integrate-into-an-existing-project) and [here](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/ghdocs/README.md#step-3-add-documentcardbasic-component).)

**Comment by mikewheaton** [comment]
Sorry for the confusion here. @c-w is right that the `<Fabric>` component is required to be a parent of all other Fabric React components, and we haven't documented this. I'm going to rename this issue so that we can update the documentation.

**Comment by cschleiden** [comment]
I think we are tracking this as part of https://github.com/OfficeDev/office-ui-fabric-react/issues/351
